resources/aws/route.go

```diff
@@ -0,0 +1,42 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/giantswarm/microerror"
+)
+
+type Route struct {
+	RouteTableID         string
+	DestinationCidrBlock string
+	VpcID                string
+	AWSEntity
+}
+
+func (r *Route) CreateIfNotExists() (bool, error) {
+	return false, microerror.Mask(notImplementedMethodError)
```

Comment:
Routes are bound to the `RouteTableID`, the ID changes with each cluster creation - therefore the route cannot easily be reused. (Could be if the route table and route both weren't deleted? not certain)

Reply:
It's fine if we can't reuse the route but please make sure all the peering routes are deleted. We can't delete the VPC until all sub resources are deleted so it's easy to have orphan resources after cluster deletion.

Reply:
I did this, but it got fairly ugly. `Route` can not be tagged and it does not have `DescribeRouteInput`. The best I could come up with was fetching the Routes from the `RouteTable` and iterating through them.